meson.build

```
@@ -3,7 +3,12 @@ project('io.elementary.calendar',
     version: '5.1.1'
 )
 
-add_project_arguments('-DGETTEXT_PACKAGE="@0@"'.format(meson.project_name()), language: 'c')
+add_project_arguments([
+    '-DGETTEXT_PACKAGE="@0@"'.format(meson.project_name()),
+    '-DHANDY_USE_UNSTABLE_API'
+    ],
+    language: 'c'
+)
```

API is stable in 0.90.0

```suggestion
add_project_arguments('-DGETTEXT_PACKAGE="@0@"'.format(meson.project_name()), language: 'c')
```

---

Yep, you're right. Thanks for pointing out.